.github/workflows/pr_shelter.yml

```diff
@@ -147,7 +148,7 @@ jobs:
       if: ${{ contains(matrix.sgx, 'SGX2') }}
       run: |
         docker cp /etc/sgx_default_qcnl.conf  $inclavare_test:/etc/;
-        docker exec $inclavare_test bash -c 'cd /opt/enclave-tls/bin/ && ./enclave-tls-server -l debug -a sgx_la -t wolfssl_sgx -c wolfcrypt -i 127.0.0.1 -p ${{ steps.random-port-generator3.outputs.random-port }}' &
+        docker exec $inclavare_test bash -c 'cd /opt/enclave-tls/samples && ./enclave-tls-server -l debug -a sgx_la -t wolfssl_sgx -c wolfcrypt -i 127.0.0.1 -p ${{ steps.random-port-generator3.outputs.random-port }}' &
```

Comment:
ditto

Reply:
Change the path to install path then if any error, we can copy the real environment.

Reply:
> Change the path to install path then if any error, we can copy the real environment.

You need to mkdir -p /usr/share/enclave-tls in the docker container at first, Thanks.